src/main/webapp/style.css

```diff
@@ -8,56 +8,570 @@ body {
   color: #353535;
 }
 
-header {
-  align-items: center;
-  background-color: #b3ffb3;
-  display: flex;
-  justify-content: space-between;
-  padding: 50px 5%;
-}
 
-/*
-  Main Content Container Styling
-*/
-.main-container {
-  display: grid;
-  column-gap: 2%;
-  grid-template-columns: [map] .5fr [results] .35fr [filters] .15fr;
-  grid-template-rows: 75vh;
-  justify-content: space-around;
-  margin: 2%;
-}
+@media screen 
+  and (min-device-width: 768px) 
+  and (max-device-width: 1200px) 
+  and (-webkit-min-device-pixel-ratio: 1) { 
 
-/*
-  Map Content Container Styling
-*/
-.map-container {
-  background-color: #e5e5e5;
-  border-radius: 30px;
-}
+    header {
+      align-items: center;
+      background-color: #b3ffb3;
```

Comment:
If you're interested (and only if you're interested), you could set these main colors to variables in the root of the css [file](https://stackoverflow.com/questions/1875852/how-can-i-define-colors-as-variables-in-css). This would make the document a little bit more readable. I'm not sure if this is good form or not but just a thought!

Reply:
I think I'll leave them as-is for now but may end up changing them later once I see if I can change around the screen breakpoints. Good suggestion though, never thought of that.

Reply:
This will also prevent having to repeat the styles for the same elements. @media selectors should work like inheritance, in that one only needs to redefine the changing values, not the entire suite. I think this will make like easier going forward. 

What do you mean changing around the screen breakpoints? The values for min-device-width etc?

Reply:
Yeah, I'm still playing around with the widths. I realize about the inheritance now, I wasn't thinking about it right, will change.